projects/core/components/notification/notification.style.less

```diff
@@ -81,4 +91,16 @@
     flex: 1;
     word-break: break-word;
     color: var(--tui-text-01);
+    text-align: left;
+
+    &::ng-deep tui-svg:last-child {
```

Comment:
Let's not add styles for that. Notification's content is a responsibility of the person using it. They can either use display: flex and gap or float: right and margin.

Reply:
fixed